WordPress/Classes/Services/MediaService.m

```diff
@@ -434,11 +434,13 @@ - (NSError *)customMediaUploadError:(NSError *)error remote:(id <MediaServiceRem
             }
         }
     }
+
     if (customErrorMessage) {
-        NSMutableDictionary *userInfo = [[NSMutableDictionary alloc] init];
+        NSMutableDictionary *userInfo = [error.userInfo mutableCopy];
```

Comment:
This reverts the fix [introduced here](https://github.com/wordpress-mobile/WordPress-iOS/pull/17488/files#diff-24c20c98b279ad5a4856285338d0e4a1a25847f7be842379bd954844098c685dR438) in favor of passing all `UserInfo` values up to the new setter, which could cherry pick more if needed in the future.